core/models.py

```diff
@@ -1 +1,11 @@
+from django.db import models
 # Create your models here.
+
+
+class NextMeeting(models.Model):
+    title = models.CharField(max_length=50)
+    description = models.TextField()
+    date_of_meeting = models.DateTimeField("Data spotkania")
```

Comment:
why did you put "Data spotkania" and not "Opis"?

Reply:
because date of meeting doesn't need a description (i think so)